_config.yml

```diff
@@ -70,6 +70,13 @@ aux_links:
 # Makes Aux links open in a new tab. Default is false
 aux_links_new_tab: false
 
+# Dropdown settings
+drop_title: "Dropdown"
+
+drop_links:
```

Comment:
Eventually we should have a nested for loop to support more than one dropdown button.

Reply:
Yes, if our site requires it we can implement it.